tool/tsh/kube.go

```diff
@@ -96,18 +96,12 @@ func (c *kubeCredentialsCommand) run(cf *CLIConf) error {
 		// a new one.
 	}
 
-	activeRequests, err := k.ActiveRequests()
-	if err != nil {
-		return trace.Wrap(err)
-	}
-
 	log.Debugf("Requesting TLS cert for kubernetes cluster %q", c.kubeCluster)
 	err = client.RetryWithRelogin(cf.Context, tc, func() error {
 		var err error
 		k, err = tc.IssueUserCertsWithMFA(cf.Context, client.ReissueParams{
 			RouteToCluster:    c.teleportCluster,
 			KubernetesCluster: c.kubeCluster,
-			AccessRequests:    activeRequests.AccessRequests,
```

Comment:
Hm, is this handled by IssueUserCertsWithMFA function now? I may have missed this, just making sure.

Reply:
Yep, should be covered by https://github.com/gravitational/teleport/blob/a9bf87d940bec0e1ca58caf155b684bbbe85ccd2/lib/client/client.go#L400-L406 which is called from both `IssueUserCertsWithMFA` and `ReissueUserCerts`.